services/orderService.js

```diff
@@ -0,0 +1,11 @@
+import OrderModel from "../models/order.js"
+
+const addOrderService = async (createdAt, product, quantity, price, user) => {
```

Comment:
you don't need to postfix metods/functions with Service, 
simple createOrder, getOrder ...